sycl/test/lit.cfg.py

```diff
@@ -41,6 +41,7 @@
 # Configure LD_LIBRARY_PATH or corresponding os-specific alternatives
 if platform.system() == "Linux":
     config.available_features.add('linux')
+    config.available_features.add('libc++')
```

Comment:
This line unconditionally disables ABI tests. Instead, it is better to add new flag [here](https://github.com/intel/llvm/blob/sycl/sycl/test/lit.site.cfg.py.in) for `@SYCL_USE_LIBCXX@` and conditionally enable `libc++` feature if the flag presents.

Reply:
Hi, @alexbatashev
Just updated patch, could you help me review if it aligns with your expectation?
By the way, I changed the feature name from "libc++" to "libcxx" which aims to align with "SYCL_USE_LIBCXX".
Thanks very much.

Reply:
Yes, it's fine now.